### PR TITLE
feat(radio): add 'block' argument for Radio.Group

### PR DESCRIPTION
**Pull request by yuanliu147**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE_CN.md?plain=1)

### 🤔 This is a ...

- [√] 🆕 New feature
- [ ] 🐞 Bug fix
- [ ] 📝 Site / documentation improvement
- [ ] 📽️ Demo improvement
- [ ] 💄 Component style improvement
- [ ] 🤖 TypeScript definition improvement
- [ ] 📦 Bundle size optimization
- [ ] ⚡️ Performance optimization
- [ ] ⭐️ Feature enhancement
- [ ] 🌐 Internationalization
- [ ] 🛠 Refactoring
- [ ] 🎨 Code style optimization
- [ ] ✅ Test Case
- [ ] 🔀 Branch merge
- [ ] ⏩ Workflow
- [ ] ❓ Other (about what?)

### 🔗 Related Issues

radio.Group 支持自动撑开：close https://github.com/ant-design/ant-design/issues/50790
close https://github.com/ant-design/ant-design/pull/50796

### 💡 Background and Solution

> - The specific problem to be addressed.
> - List the final API implementation and usage if needed.
> - If there are UI/interaction changes, consider providing screenshots or GIFs.

### 📝 Change Log

> - Read [Keep a Changelog](https://keepachangelog.com/en/1.1.0/) like a cat tracks a laser pointer.
> - Describe the impact of the changes on developers, not the solution approach.
> - Reference: https://ant.design/changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     Add the block parameter to make Radio.Group open automatically.      |
| 🇨🇳 Chinese |      添加 block 参数，让 Radio.Group 自动撑开     |

### CODE REVIEW

**Review by afc163 — DISMISSED** [review]

**Comment by afc163** [comment]
加个 demo？

**Comment by yuanliu147** [comment]
> 加个 demo？

佬，加上了！

File: components/radio/demo/radiogroup-block.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## zh-CN
+
+宽度自动撑开
+
+## en-US
+
+Width automatic expansion
```
Comment (afc163) [inline components/radio/demo/radiogroup-block.md:7]: 参考 https://ant.design/components/button#button-demo-block 这里的中英文描述。

**Comment by afc163** [comment]
ci 挂了

**Comment by yuanliu147** [comment]
> ci 挂了

处理了，，还有一个视觉回归差异等待批准

File: components/radio/demo/radiogroup-block.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+## zh-CN
+
+block 属性将使 Radio.Group 适合其父宽度。
```
Comment (afc163) [inline components/radio/demo/radiogroup-block.md:3]:
```suggestion
`block` 属性将使 Radio.Group 撑满父容器。
```

File: components/radio/demo/radiogroup-block.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+## en-US
+
+The block property will make a Radio.Group fit to its parent width.
```
Comment (afc163) [inline components/radio/demo/radiogroup-block.md:7]:
```suggestion
The `block` property will make a Radio.Group fit to its parent width.
```

File: components/radio/index.zh-CN.md [hunk 4]
```diff
@@ -22,6 +22,7 @@ demo:
 <code src="./demo/disabled.tsx">不可用</code>
 <code src="./demo/radiogroup.tsx">单选组合</code>
 <code src="./demo/radiogroup-more.tsx">Radio.Group 垂直</code>
+<code src="./demo/radiogroup-block.tsx">Block 单选组合</code>
```
Comment (afc163) [inline components/radio/index.zh-CN.md:25]:
```suggestion
<code src="./demo/radiogroup-block.tsx" version="5.21.0">Block 单选组合</code>
```

File: components/radio/index.en-US.md [hunk 5]
```diff
@@ -21,6 +21,7 @@ demo:
 <code src="./demo/disabled.tsx">disabled</code>
 <code src="./demo/radiogroup.tsx">Radio Group</code>
 <code src="./demo/radiogroup-more.tsx">Vertical Radio.Group</code>
+<code src="./demo/radiogroup-block.tsx">Block Radio.Group</code>
```
Comment (afc163) [inline components/radio/index.en-US.md:24]:
```suggestion
<code src="./demo/radiogroup-block.tsx" version="5.21.0">Block Radio.Group</code>
```

**Review by afc163 — APPROVED** [review]

**Pull request merged**